### PR TITLE
messing with main as example

### PR DESCRIPTION
**Pull request by dupuisjames**

### CODE REVIEW

File: DispatchMayhem/Assets/Scenes/Main.unity [hunk 1]
```diff
@@ -269,12 +269,16 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 4867834046800562, guid: e6b749c6d877f4c19a5a5c3c0783d53b, type: 3}
       propertyPath: m_LocalScale.x
-      value: 500
+      value: 6000
```
Comment (dupuisjames) [inline DispatchMayhem/Assets/Scenes/Main.unity]: why did you remove my 500 and put 6000?

**Review by dupuisjames — COMMENTED** [review]
dcdvdcv

File: DispatchMayhem/Assets/Scenes/Main.unity (hunk 1 above)
Comment (dupuisjames) [reply DispatchMayhem/Assets/Scenes/Main.unity]: i thought this was the better value

**Review by mjes87 — APPROVED** [review]

**Review by SyTuck — APPROVED** [review]
looks fine by me
